### PR TITLE
add customization option for message reaction inserter

### PR DESCRIPTION
**Pull request by macovsky**
what do you think about such cosmetic change?

IMO this way it looks better since user sees up to three avatars anyways

### CODE REVIEW

**Comment by zevlg** [comment]
There are few things to consider about this modification:
1) For blind telega users it is very useful to see number of reactions before taking any action, so this must be customizable after all
2) Recent reaction senders are not always available, so you won't be able to distinguish single reaction from double reaction
3) User might customize `telega-avatar-text-function`, so in nox-emacs he might not see recent reactions avatars at all even if the list of recent reaction senders is available

It is better to advice `telega-ins--msg-reaction` to implement custom behaviour you need

**Comment by macovsky** [comment]
> There are few things to consider about this modification:
> 
> 1. For blind telega users it is very useful to see number of reactions before taking any action, so this must be customizable after all
> 2. Recent reaction senders are not always available, so you won't be able to distinguish single reaction from double reaction
> 3. User might customize `telega-avatar-text-function`, so in nox-emacs he might not see recent reactions avatars at all even if the list of recent reaction senders is available

good points of course, thanks for going into details!

> It is better to advice `telega-ins--msg-reaction` to implement custom behaviour you need

will try to make it!

**Comment by macovsky** [comment]
@zevlg added a customization option like you adviced

File: telega-customize.el [hunk 1]
```diff
@@ -1125,6 +1125,11 @@ See `telega-ins--message' for NO-HEADER argument."
   :type 'function
   :group 'telega-inserter)
 
+(defcustom telega-inserter-for-msg-reaction 'telega-ins--msg-reaction
+  "Inserter for message reaction."
```
Comment (zevlg) [inline telega-customize.el:1129]: Add please `:package-version` tag to this user option

**Review by zevlg — CHANGES_REQUESTED** [review]
This looks decent!

File: telega-customize.el [hunk 2]
```diff
@@ -1125,6 +1125,12 @@ See `telega-ins--message' for NO-HEADER argument."
...
+  :package-version '(telega . "0.8.255")
```
[the remark this answers is not in the document]
Comment (zevlg) [reply telega-customize.el:1130]: No, unless you change `telege-version` in the telega.el, put last `telega-version` , i.e. `0.8.254`

[the remark this answers is not in the document]
Comment (macovsky) [reply telega-customize.el]: got it, fixed!

**Pull request merged**